app/_hub/kong-inc/mocking/index.md

```diff
@@ -0,0 +1,897 @@
+available on YouTube.
+</div>
+
+Prerequisites:
+
+- {{site.ee_product_name}} environment with the Dev Portal enabled on at least one workspace. See
+  [enable the Dev Portal](/enterprise/latest/developer-portal/enable-dev-portal/) using `kong.conf`. Also refer to
+  the instructions for your environment:
+  - [Docker](/enterprise/latest/deployment/installation/docker/#step-7-optional-enable-the-dev-portal)
+  - [Ubuntu](/enterprise/latest/deployment/installation/ubuntu/#optional-enable-the-dev-portal)
+  - [CentOS](/enterprise/latest/deployment/installation/centos/#optional-enable-the-dev-portal)
+  - [Amazon Linux 1](/enterprise/latest/deployment/installation/amazon-linux/#optional-enable-the-dev-portal)
+  - [Amazon Linux 2](/enterprise/latest/deployment/installation/amazon-linux-2/#optional-enable-the-dev-portal)
+  - [RHEL](/enterprise/latest/deployment/installation/rhel/#optional-enable-the-dev-portal)
+
+- An Open API Specification (`yaml` or `json`) that has at least one API method with an
+  embedded example response. Multiple examples within a spec are supported. See the
+  [Stock API spec example](#stock-spec).
+
+Tutorial steps:
+
+1. Deploy the example OAS spec that contains mocked responses to the [Dev Portal](#deploy-spec-portal) or
+   [Insomnia](#deploy-spec-insomnia).
+2. Create the [Stock service](#create-stock-service).
+3. Create the [get stock quote route](#create-stock-quote-route).
+4. Enable the [Mocking plugin](#enable-mock-plugin) on the get stock quote route.
+5. Enable the [CORS plugin](#enable-cors-plugin) on the get stock quote route.
+6. [Test the mocked response](#testing123) from the Dev Portal, Insomnia, or the command line.
+7. When you've completed your API mock testing, [disable the Mocking plugin and update the Service URL](#post-test).
+
+### Step 1. Deploy the Stock API example spec
+
+Deploy the [Stock API spec example](#stock-spec) to either the [Dev Portal](#deploy-spec-portal) or
+[Insomnia](#deploy-spec-insomnia).
+
+#### Deploy a spec to the Dev Portal {#deploy-spec-portal}
+
+Follow these steps to deploy a spec to the Dev Portal using Kong Manager. You can
+copy and paste the `stock-01.json` example file into the Dev Portal using Editor Mode.
+
+![Dev Portal Specs](/assets/images/docs/dev-portal/stock-spec-mock-example.png)
+
+1. Open Editor Mode and click **New File**.
+2. Name the file `stock-01.json`.
+3. Copy and paste the contents in the [example](#stock-spec) into the new file.
+
+Alternatively, you can also use the [Portal Files API](/enterprise/latest/developer-portal/files-api/#post-a-content-file)
+to upload a spec to the Dev Portal.
+
+#### Deploy a spec to Insomnia {#deploy-spec-insomnia}
+
+1. From the Insomnia dashboard, click **Create** > **Import from File** and select the
+   `stock-0.1.json` file.
+
+   ![Insomnia Dashboard Import File](/assets/images/docs/insomnia/insomnia-import-spec.png)
+
+   You are prompted to choose an import option.
```

Comment:
```suggestion
```
Re: recent conversations with the doc team, we should avoid this type of language. Instead, edit the following line to say "Choose **Design Document** from the import options", if you have to include this info.